CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ cmake_minimum_required(VERSION 3.14)
 project(lei3d)
 
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON CACHE INTERNAL "")
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
Good for us to specify to the build type. We'll want to make a note of this change when we start trying to get release executables.